README.md

```diff
@@ -27,39 +33,188 @@
   </a>
 </p>
 
-## Backstory
-
-...
-
 ## Key features
 
 - 🍱 Reliable serialization and deserialization
 - 🔐 Type safety with autocompletion
 - 🐾 Negligible runtime footprint
 - 💫 Framework agnostic
 
-## Example
+
+## Backstory
+
+At [Blitz](https://github.com/blitz-js/blitz), we have struggled with the limitations of JSON. Modern databases often store complex types, like Date objects, but it impossible to easily `stringify` your responses without converting any object types to strings. Moreover, you often have to keep track of which fields you need to convert back to objects in your client!
+
+Superjson solves these issues by providing a thin wrapper over `JSON.stringify` and `JSON.parse`. Look at the difference between manually converting invalid fields, and using Superjson to handle this for you:
+
+
+```js
```

Comment:
Nice idea, but I fear this example is a bit lengthy. Could ellipsize unnecessary lines?